src/main/java/bot/command/instruction/execute/StorageWriting.java

```diff
@@ -0,0 +1,21 @@
+package bot.command.instruction.execute;
+
+import bot.Storage;
+import bot.Ui;
+
+/**
+ * An interface that represents Instructions that
+ * need to write to the Storage
+ */
+public interface StorageWriting<T> {
```

Comment:
Good modularisation of the abstract functionalities, of a storage drivers into an interface. This allows various classes with different implementations to implement these abstract operations.